src/IW/Sync/Search.hs

```diff
@@ -28,70 +35,117 @@ import qualified Data.Text as T
 import qualified Data.Vector as V
 
 
--- | Fetch all repositories built with the Haskell language.
-fetchAllHaskellRepos
-    :: ( MonadIO m
+githubSearch
```

Comment:
I think this function was previously called `liftGitHubSearchToApp`. In that case, it misses documentation.

Reply:
Also, I believe that this function should be separated into two functions. I think it still makes sense to kepp `liftGitHubSearchToApp` with slight modifications. `lift*` function will perform GitHub query and lift it to our `Monad` and perform some logging. And `githubSearch` will just construct a query from arguments, log query and call `liftGitHubSearchToApp`